stream-chat-android-compose/src/main/res/drawable/stream_compose_ic_message_pinned.xml

```diff
@@ -2,9 +2,11 @@
     android:width="16dp"
     android:height="16dp"
     android:viewportWidth="16"
-    android:viewportHeight="16">
-  <path
-      android:pathData="M10.694,1.529C10.5304,1.3653 10.2937,1.2981 10.0685,1.3513C9.8433,1.4044 9.6616,1.5704 9.5884,1.7899L9.1661,3.0568L6.7203,5.5027C5.1281,5.142 3.6,5.7902 2.1956,7.1946C1.9348,7.4553 1.9348,7.8781 2.1956,8.1388L4.5563,10.4995L3.1399,11.9159C2.8791,12.1767 2.8791,12.5994 3.1399,12.8601C3.4007,13.1209 3.8234,13.1209 4.0842,12.8601L5.5005,11.4438L7.8612,13.8045C8.1219,14.0652 8.5447,14.0652 8.8055,13.8045C10.2099,12.4 10.858,10.872 10.4973,9.2798L12.9432,6.8339L14.2101,6.4116C14.4296,6.3384 14.5956,6.1568 14.6488,5.9316C14.7019,5.7064 14.6347,5.4697 14.4711,5.306L10.694,1.529ZM10.3832,3.6286L10.5137,3.2372L12.7629,5.4864L12.3714,5.6169C12.2731,5.6497 12.1837,5.7049 12.1104,5.7782L9.2776,8.611C9.0987,8.7898 9.0363,9.0543 9.1163,9.2942C9.4345,10.2489 9.2382,11.2565 8.3129,12.3676L3.6324,7.6872C4.7435,6.7618 5.7511,6.5655 6.7058,6.8838C6.9457,6.9637 7.2102,6.9013 7.3891,6.7224L10.2219,3.8896C10.2951,3.8163 10.3504,3.727 10.3832,3.6286Z"
-      android:fillColor="#72767E"
-      android:fillType="evenOdd"/>
+    android:viewportHeight="16"
```

Comment:
Should we change the dimensions to 24x24 here as well?

Reply:
Resolved outside of this thread. 

Result: Not all icons have to be 24x24, there are other standardized sizes such as 16x16 for small icons and 128x128 for placeholders